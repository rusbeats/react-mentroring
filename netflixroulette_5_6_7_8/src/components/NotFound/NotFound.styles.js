import styled from 'styled-components';

export const StyledNotFound = styled.div`
    height: 70vh;
    background: url('/static/404.png') no-repeat center;
`;
