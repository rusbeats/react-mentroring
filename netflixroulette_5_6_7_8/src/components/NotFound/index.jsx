import React from 'react';
import { StyledNotFound } from './NotFound.styles';

// Changed to an image for '404' message
const NotFound = () => {
    return <StyledNotFound />;
};

StyledNotFound.displayName = 'NotFound';

export default NotFound;
