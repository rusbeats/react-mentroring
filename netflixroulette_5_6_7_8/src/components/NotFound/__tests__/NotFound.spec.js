import React from 'react';
import { shallow } from 'enzyme';
import NotFound from '../index';

describe('NotFound', () => {
    it('should render correctly (snapshot)', () => {
        const component = shallow(<NotFound />);
        expect(component).toMatchSnapshot();
    });
});
