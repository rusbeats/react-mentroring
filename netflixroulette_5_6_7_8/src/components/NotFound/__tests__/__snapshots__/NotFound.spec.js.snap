// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should render correctly (snapshot) 1`] = `<NotFound />`;
